src/main/webapp/app/exercises/text/assess-new/textblock-feedback-editor/textblock-feedback-editor.component.ts

```diff
@@ -42,6 +42,16 @@ export class TextblockFeedbackEditorComponent implements AfterViewInit {
      */
     ngAfterViewInit(): void {
         this.textareaElement = this.textareaRef.nativeElement as HTMLTextAreaElement;
+        setTimeout(() => this.textareaOnloadSize());
+    }
+
+    /**
+     * Increase size of text area on page load to a max of 55px, depending on content
+     */
+    textareaOnloadSize(): void {
+        this.textareaElement.style.height = '0px';
+        this.textareaElement.style.height = `${this.textareaElement.scrollHeight}px`;
+        this.textareaElement.style.height = this.textareaElement.scrollHeight > 55 ? '55px' : `${this.textareaElement.scrollHeight}px`;
```

Comment:
Is this the desired behavior?
Wouldn't it be preferred to have all elements at the needed height and not require scrolling here?

Reply:
I used @filip-gregurevic suggestion (see the comment above)j, to let it expand to a certain height and if less text is entered (<2 lines), then only this (1 line e.g.) is shown. Would you prefer a different behaviour?

Reply:
So you are using this more than I am, so I am open to everything. :)
My personal preference is to reduce scrolling within scrollable content as much as possible.

Reply:
That's true.
As the text area expands (and stays expanded) while editing and only reduces in size on reopening, scrolling is reduced anyways. Though when reopening and decreasing the text box sizes, lets everyting appear very structured and clear look. So I think this is a nice compromise :).